scripts/v2/wait-for-operator-ready.sh

```diff
@@ -60,7 +64,7 @@ kubectl wait --for=condition=ready --timeout=3m pod -n "$OPERATOR_NAMESPACE" -l
 echo "Waiting for CRD cabundle..."
 export -f all_crds_have_cabundle
 export -f wait_for_crds_cabundle
-timeout 2m bash -c wait_for_crds_cabundle
+timeout 8m bash -c wait_for_crds_cabundle
```

Comment:
Isn't 8 mins from 2 too high? 
Just thinking about if something else goes haywire then it waits for 4x longer than it used to.

Reply:
I incremented from 2m by 1m steps; things finally worked end to end at 7m, so I added one more for a little headroom.

Totally agree that this should be no higher than necessary.